Solana: Implement a CLI Command for Program Deployment (#1542)
hyperledger/solang: docs/running.rst

  There is only supported on Solana.

> Submitting extrinsics to Solana on-chain

Reviewer: extrinsics is a polkadot term 

Running programs on-chain?

_________________________________________________

> 
>   solang solana [SUBCOMMAND] [OPTIONS]... <PROGRAM_LOCATION>
> 
> This means that the command line is ``solang solana`` followed by a subcommand 
> followed by any of the options described below and then the program location.

Reviewer 2: ```suggestion

The command line syntax for interacting with a program deployed on Solana is as follows:

  solang solana [SUBCOMMAND] [OPTIONS]... <PROGRAM_LOCATION>

It consists of a subcommand followed by its options, both of which are described below. The last argument is the program location.
```

> General Options (for all subcommands):

Reviewer 2: ```suggestion
General Options (for all subcommands):
++++++++++++++++++++++++++++++++
```

\-\-output-json
  Specifies whether to export the call output in JSON format.

\-\-help, -h
  This displays a short description of all the options.

> Subcommands:

Reviewer 2: ```suggestion
Subcommands:
++++++++++++
```

  solang solana deploy [OPTIONS]... <PROGRAM_LOCATION>

Deploys a program to Solana.

Options specific to the ``deploy`` subcommand:

\-\-verbose, -v
  Specifies whether to display verbose program deployment information
  (Conflicts with ``--output-json``).

> 

Reviewer 2: `deploy` is a subcommand, but the way you wrote it makes it look like it is another command. Instead of starting the section with the command, write first about what it does and its options, then give an usage example.

Author: good point
